### ISSUE TITLE
Draft new H4CC pages on presidential monthly filing and host and inaugural committee reporting

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2880]
**What we are after:**
Presidential campaigns need guidance too. In past issues, we've identified resources for it and made a list of new pages we need to create. This ticket is to create new pages that will go in the candidate filing bucket as child pages of https://www.fec.gov/help-candidates-and-committees/filing-reports/.

**Past work:** #2651 (resources gathered), #2769 (new pages listed), #2807 (work plan for new pages)

**To do**
- [x]  Find approved language (will need to go through Guides and other sources for convention page)
- [x]   Draft content in a Word doc for each page; note source of content
- [x]   Circulate to  Information Division and content team for review and comment.

**Future work**
-NEXT Ticket: Circulate to OGC along with the other new pages in issue #2878 and #2879 
-When OGC comments are back, create and publish the pages, and add links to collection page for candidates.

**Comment by dorothyyeager** [comment]
A thought that just occurred to me. We realized we need to do something for host committee reporting. But we'll need to keep in mind that unlike everything else, host committees would fall more under "Other filers" since there's no candidate involved. cc: @kathycarothers

**Comment by kathycarothers** [comment]
Starting to draft the monthly page in Word.

**Comment by kathycarothers** [comment]
We do have this page on inaugurals committees https://www.fec.gov/press/resources-journalists/inaugural-committees/ . Do we want to link it from another page? @dorothyyeager

**Comment by dorothyyeager** [comment]
Hmmm this page would actually work really well for our "inaugural committee" reporting page purposes. The only other thought I have on it is that it does lack info on making donations to inaugural committees (there's a foreign national prohibition) and it doesn't explain how an inaugural committee registers. So if we do bring it over into the H4CC format, we'd want to add those items. 

@AmyKort what do you think - do you think we could just cross-link the press office page in the Filing candidate reports section? Or should we approach the Press Office and ask if we can convert this into a H4CC page? Or have pages living both places?

**Comment by dorothyyeager** [comment]
Also - same thoughts on this page - https://www.fec.gov/press/resources-journalists/convention-committees-and-host-committees/. Maybe because there are so few host and inaugural committees, we don't need to put a lot of effort into it. 
cc: @bmathesonFEC @AmyKort @kathycarothers @ChristianHilland  what do you think about Kathy and my thoughts (see above) on the pages for convention and host committees moving over or being linked in H4CC?

**Comment by AmyKort** [comment]
I think reaching out to the Press Office is a great idea, @dorothyyeager . I think we should avoid having the information in two places, and I'm happy to link to the press page from H4CC. But, the press folks might have a really good perspective on this, since I think they probably end up answering a lot of the questions that come in on this topic.

**Comment by kathycarothers** [comment]
I have circulated the presidential monthly page to the content team. I am currently waiting on the other two pages until we get a chance to talk to Press.

**Comment by kathycarothers** [comment]
Talked to Christian in Press and they do not have a problem with the following pages being moved over to the Candidate section:

https://www.fec.gov/press/resources-journalists/convention-committees-and-host-committees/ and 
https://www.fec.gov/press/resources-journalists/inaugural-committees/

**Comment by AmyKort** [comment]
Thanks @kathycarothers ! That's a great solution.

**Comment by kathycarothers** [comment]
I have sent the draft host and inaugural committees pages to the content team for review. Deadline is tomorrow COB.

**Comment by kathycarothers** [comment]
Deadline has past so this ticket is being closed.
